gpg-expiration.py

```diff
@@ -49,8 +51,21 @@ def load_key(key_id):
             raise ValueError('Unable to load key', key_id) from ex
 
 
+def check_all(not_before):
+    return_value = 0
+    with gpg.Context() as ctx:
+        pubkeys = ctx.keylist(pattern=None, secret=False)
+        for key in list(pubkeys):
```

Comment:
I think we want to make sure we find at least one key here - silently succeeding in the case where the keyring is empty would be bad!